index.bs

```diff
@@ -743,14 +782,31 @@ partial interface HTMLIFrameElement {
     <div class="algorithm" data-algorithm="parse-header">
     Given a string (|value|) and an [=origin=] (|origin|), this algorithm will
     return a <a>declared permissions policy</a>.
+    1. Let |parsed header| be the result of parsing |policy| as a dictionary.
```

Comment:
Where does policy come from? Also, I think this algorithm should take a SH dictionary. And elsewhere we'd use https://fetch.spec.whatwg.org/#concept-header-list-get-structured-header to get it from a header list.

Reply:
Oops -- |policy| should have been |value| here.

And we can take a dictionary and defer to fetch for the SH parsing, thanks.